tests/bedrock/test_bedrock_autolog.py

```diff
@@ -294,6 +294,81 @@ def test_bedrock_autolog_invoke_model_capture_exception():
     assert span.events[0].attributes["exception.message"].startswith("Unable to locate credentials")
 
 
+def test_bedrock_autolog_invoke_model_stream():
+    mlflow.bedrock.autolog()
+
+    client = boto3.client("bedrock-runtime", region_name="us-west-2")
+    request_body = json.dumps(_ANTHROPIC_REQUEST)
+
+    dummy_chunks = [
+        {
+            "type": "message_start",
+            "message": {
+                "id": "123",
+                "type": "message",
+                "role": "assistant",
+                "model": "claude-3-5-sonnet-20241022",
+                "content": [],
+                "stop_reason": None,
+                "usage": {"input_tokens": 8, "output_tokens": 1},
+            },
+        },
+        {
+            "type": "content_block_start",
+            "index": 0,
+            "content_block": {"type": "text", "text": "Hello"},
+        },
+        {
+            "type": "content_block_delta",
```

Comment:
nit: Should we have >1 stream entry for the `content_block_delta` just to ensure future protection coverage for consumption of repeated message block data? (Just to be safe :) )